pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = [
     { name = "Adrian D'Alessandro", email = "a.dalessandro@imperial.ac.uk" },
 ]
 requires-python = ">=3.12"
-dependencies = ["django"]
+dependencies = ["django", "whitenoise", "django-stubs[compatible-mypy]"]
```

Comment:
The stubs package should only be in the development dependencies. After making that change, the three requirements files should also be updated accordingly with `pip-compile`

Reply:
`mypy` seems to fail without it?

Reply:
Yes, we'll still need it in the `dev` dependencies under `[project.optional-dependencies]` in the `pyproject.toml` file

Reply:
I can't seem to edit this from my desktop at home without reforking it - will do on Monday when back in office if it's OK!

Reply:
Ok, I'll merge this and make that change myself so it gets merged before the end of October